Components/InitiativeCard/InitiativeCard.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const Card = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid black;
+  border-radius: 10px;
+  height: 150px;
+  width: 100%;
+`;
+
+const TopLeftButton = styled.button`
+  position: absolute;
+  top: 8px;
+  left: 8px;
+  background-color: transparent;
+  border: none;
+  font-size: 16px;
+  cursor: pointer;
+`;
+
+const Modal = styled.div`
+  position: absolute;
+  background-color: white;
+  border: 1px solid gray;
+  padding: 16px;
+  border-radius: 10px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 10;
+`;
+
+const Button = styled.button`
+  margin: 4px;
+  padding: 8px 12px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+
+  &:nth-child(1) {
+    background-color: #6c757d;
+    color: white;
+  }
+
+  &:nth-child(2) {
+    background-color: #6c757d;
+    color: white;
+  }
+
+  &:nth-child(3) {
+    background-color: #c8c8c8;
+    color: black;
+  }
+`;
+
+const Tag = styled.span`
+  background: #a8a8a8;
+  border-radius: 4px;
+  padding: 4px 8px;
+  margin: 4px;
+  font-size: 12px;
+`;
+
+export default function InitiativeCard({ title, tags, deadline }) {
+  const [isModalOpen, setModalOpen] = useState(false);
+
+  return (
+    <>
+      <Card>
+        <TopLeftButton onClick={() => setModalOpen(true)}>...</TopLeftButton>
+
+        <h3>{title}</h3>
+        <div>
+          {tags.map((tag, index) => (
+            <Tag key={index}>{tag}</Tag>
```

Comment:
You are using the index of the array here for your tags. This is an anti pattern. In fact, if you would not assign a key value at all, React would use the array index under the hood. There is a very important quality that the key values need to fulfill: 
> The key needs to be A: unique and B: be coupled to the respective element.

On the first glance both requirements are met when you use the index, but if you think about it, the order of elements in an array can change. Therefore the index of the element can change, it is not bound to the tag itself. Therefore we cannot use the index for the key. It needs to be something from the mapped element itself. 

Since it can be expected that tags will be unique, you can use the tag iteself as the key:
```suggestion
          {tags.map((tag, index) => (
            <Tag key={tag}>{tag}</Tag>
```